packages/cardpay-sdk/sdk/token-bridge-foreign-side.ts

```diff
@@ -84,19 +102,34 @@ export default class TokenBridgeForeignSide implements ITokenBridgeForeignSide {
     });
   }
 
+  async relayTokens(txnHash: string): Promise<TransactionReceipt>;
   async relayTokens(
     tokenAddress: string,
     recipientAddress: string,
     amount: string,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<TransactionReceipt>;
+  async relayTokens(
+    tokenAddressOrTxnHash: string,
+    recipientAddress?: string,
+    amount?: string,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<TransactionReceipt> {
+    if (!recipientAddress) {
```

Comment:
ditto with the comments above about testing whether the value you actually care about is shaped like an address or txnHash instead of testing a side effect